test/e2e/flowaggregator_test.go

```diff
@@ -188,16 +210,53 @@ func TestFlowAggregator(t *testing.T) {
 	}
 
 	if v4Enabled {
-		t.Run("IPv4", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, false) })
+		t.Run("IPv4", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, false, false) })
 	}
 
 	if v6Enabled {
-		t.Run("IPv6", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, true) })
+		t.Run("IPv6", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, true, false) })
+	}
+
+}
+
+func secureConnectionHelper(t *testing.T, o tlsTestOptions) {
+	data, v4Enabled, v6Enabled, err := setupTestForFlowAggregator(t, o)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer func() {
+		teardownTest(t, data)
+		// Execute teardownFlowAggregator later than teardownTest to ensure that the log
+		// of Flow Aggregator has been exported.
+		teardownFlowAggregator(t, data)
+		if o.ingress {
+			teardownIngressNginxController(t, data)
+		}
+	}()
+
+	k8sUtils, err = NewKubernetesUtils(data)
```

Comment:
is this used anywhere?

Reply:
No. I've deleted it and updated `test/e2e/flowaggregator_test.go`